mqclient/broker_client_interface.py

```diff
@@ -106,7 +106,8 @@ class RawQueue:
     """Raw queue object, to hold queue state."""
 
     def __init__(self) -> None:
-        pass
+        # can be changed by Queue per situation, then broker client handles accordingly
+        self.connection_can_have_multiple_unacked_messages = False
```

Comment:
Can you comment on what this means, since it looks like this is currently always False?